FPMS/FPMS/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for FPMS project.
+
+Generated by 'django-admin startproject' using Django 1.10.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.10/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.10/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 't+^flxg&v596q&3n_c(#kz5&k9*symytm!-qzd9#-o85s99yrz'
```

Comment:
I realize this is for dev, but we need to set up some kind of clean/smudge filter pair or env var setup to get this from

Reply:
This is true. I didn't even see this. I'll see if I can filter this out

Reply:
Like I said, it's probably fine for dev, but we can replace it with something like "<SECRET_TOKEN>" and have clean/smudge filters set up for us to switch it. They wouldn't be too hard to write, I can make a task for it and get a mock scripted, if that's the direction we want to go

Reply:
Make it so.